services/web/pkg/config/options.go

```diff
@@ -0,0 +1,65 @@
+	HoverableQuickActions    bool            `json:"hoverableQuickActions" yaml:"hoverableQuickActions" env:"WEB_OPTION_HOVERABLE_QUICK_ACTIONS" desc:"Set this option to 'true' to hide the quick actions (buttons appearing on file rows), and only show them when the user hovers the row with his mouse. Defaults to 'false'."`
+	Routing                  Routing         `json:"routing" yaml:"routing"`
+	Upload                   Upload          `json:"upload" yaml:"upload"`
+	Editor                   Editor          `json:"editor" yaml:"editor"`
+	ContextHelpersReadMore   bool            `json:"contextHelpersReadMore" yaml:"contextHelpersReadMore" env:"WEB_OPTION_CONTEXTHELPERS_READ_MORE" desc:"Specifies whether the 'Read more' link should be displayed or not."`
+	LogoutURL                string          `json:"logoutURL" yaml:"logoutURL" env:"WEB_OPTION_LOGOUT_URL" desc:"The logout url"`
+}
+
+// AccountEditLink are the AccountEditLink options
+type AccountEditLink struct {
+	Href string `json:"href" yaml:"href" env:"WEB_OPTION_ACCOUNT_EDIT_LINK_HREF" desc:"Set a different target URL for the edit link. Make sure to prepend it with 'http(s)://'."`
+}
+
+// FeedbackLink are the feedback link options
+type FeedbackLink struct {
+	Href        string `json:"href" yaml:"href" env:"WEB_OPTION_FEEDBACKLINK_HREF" desc:"Set a different target URL for the feedback link. Make sure to prepend it with 'http(s)://'. Defaults to 'https://owncloud.com/web-design-feedback'.`
+	AriaLabel   string `json:"ariaLabel" yaml:"ariaLabel" env:"WEB_OPTION_ARIALABEL" desc:"Since the link only has an icon, you can set an e.g. screen reader accessible label. Defaults to 'ownCloud feedback survey'.`
+	Description string `json:"description" yaml:"description" env:"WEB_OPTION_DESCRIPTION" desc:"Provide any description you want to see as tooltip and as accessible description. Defaults to 'Provide your feedback: We'd like to improve the web design and would be happy to hear your feedback. Thank you! Your ownCloud team'.`
+}
+
+// Sidebar are the side bar option
+type Sidebar struct {
+	Shares SidebarShares `json:"shares" yaml:"shares"`
+}
+
+// SidebarShares are the options for the shares side bar
+type SidebarShares struct {
+	ShowAllOnLoad bool `json:"showAllOnLoad" yaml:"showAllOnLoad" env:"WEB_OPTION_SIDEBAR_SHARES_SHOW_ALL_ON_LOAD" desc:"Sets the list of (link) shares list in the sidebar to be initially expanded (default is a collapsed state, only showing the first three shares)."`
+}
+
+// Routing are the routing options
+type Routing struct {
+	IDBased bool `json:"idBased" yaml:"idBased" env:"WEB_OPTION_ROUTING_ID_BASED" desc:"Enable or disable fileIds being added to the URL. Defaults to 'true' because otherwise e.g. spaces with name clashes can't be resolved correctly. Only disable this if you can guarantee server side that spaces of the same namespace can't have name clashes."`
```

Comment:
```suggestion
	IDBased bool `json:"idBased" yaml:"idBased" env:"WEB_OPTION_ROUTING_ID_BASED" desc:"Enable or disable fileIds being added to the URL. Defaults to 'true', because otherwise spaces with name clashes cannot be resolved correctly. Note: Only disable this if you can guarantee on the server side, that spaces of the same namespace cannot have name clashes."`
```